crypto/src/main/java/org/hyperledger/besu/crypto/SignatureAlgorithmType.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.crypto;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+public class SignatureAlgorithmType {
+
+  private static final Map<String, Supplier<SignatureAlgorithm>> TYPES =
```

Comment:
Could we rename this to `SUPPORTED_ALGORITHMS`

Reply:
Yes! That name is so much better.